src/containers/account/account.js

```diff
@@ -3,26 +3,66 @@ import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import { firebaseConnect } from 'react-redux-firebase'
 import PropTypes from 'prop-types'
+import { get } from 'lodash'
 import AccountPage from '../../presentation/account/accountPage'
+import Snackbar from 'material-ui/Snackbar'
+
+const firebaseErrorCodeToFriendlyMessage = (errorCode) => {
+  switch (errorCode) {
+    case 'auth/invalid-email': return 'Invalid Email'
+    case 'auth/email-already-in-use': return 'Email already in use'
+    case 'auth/requires-recent-login': return 'Recent login required. Please sign out and back in'
+    default: return 'There was an issue changing your email. Please try again'
+  }
+}
 
 const Account = props => (
-  <AccountPage {...props} />
+  <div>
+    <AccountPage
+      {...props}
+      handleProfileChanges={(values) => {
+        props.firebase.updateProfile({ firstName: values.firstName, lastName: values.lastName });
+        props.firebase.updateEmail(values.email);
+      }}
+    />
+    <Snackbar
+      bodyStyle={{ backgroundColor: '#F44336' }}
+      open={props.account.signInError !== undefined}
```

Comment:
Shouldn't this be the error if updateProfile or updateEmails fails? Not if sign in fails.